.github/workflows/ingestion-deploy-dev.yaml

```diff
@@ -15,10 +15,10 @@ jobs:
     with:
       app-name: lamp-ingestion
       environment: dev
-      dockerfile-path: ./py_gtfs_rt_ingestion/
+      dockerfile-path: ./python_src/
       cluster: lamp
     secrets:
       aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
       aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       docker-repo: ${{ secrets.INGESTION_DOCKER_URI }}
```

Comment:
Since we're using the same Dockerfile for all applications, should probably use the same `docker-repo` for both apps.